src/actions/guides/deploying/heroku.cr

```diff
@@ -141,12 +141,10 @@ class Guides::Deploying::Heroku < GuideAction
     we can set the `SECRET_KEY_BASE` environment variable with that key:
 
     ```bash
-    $ lucky gen.secret_key
-    <GENERATED_SECRET_KEY>
-    $ heroku config:set SECRET_KEY_BASE=<GENERATED_SECRET_KEY>
+    $ heroku config:set SECRET_KEY_BASE=$(lucky gen.secret_key)
     ```
 
-    ### `SEND_GRID_KEY` (optional)
+    ### `SEND_GRID_KEY` (set to "unused" if not sending emails)
```

Comment:
Yeah! I think this line alone will reduce a ton of questions around deployment errors